src/VercelEnvVariabler.ts

```diff
@@ -238,14 +243,18 @@ export default class VercelEnvVariabler {
         gitBranch: string | undefined;
     }) {
         const existingVariable = Object.values(existingVariables)[0]; // They are all actually the same
+        const gitBranchInfoTextSnippet = gitBranch
+            ? ` on git branch '${gitBranch}'`
+            : "";
```

Comment:
👍

Reply:
... also refactored the git branch logging... you might want to a brief look 👀